build/gen_stub.php

```diff
@@ -2404,13 +2404,23 @@ function parseProperty(
         }
     }
 
+    $default = $property->default;
+    if ($property instanceof Node\Param) {
+        $name = $property->var->name;
+        if ($property->flags & Stmt\Class_::MODIFIER_READONLY) {
+            $default = null;
```

Comment:
It would make more sense for me if an exception would be thrown when a default value is provided

Reply:
Why? The default is still used for the arginfo.

Reply:
Ah, I see! Sorry, I wasn't aware that optional promoted readonly property parameters are legal, and the default values are only used for parameter reflection. Then feel free to ignore this comment :)